artem/Interactive gallary/8 - JS gallary/js/script.js

```diff
@@ -0,0 +1,29 @@
+const prevButton = document.querySelector(".prev");
+const nextButton = document.querySelector(".next");
+const gallaryImgs = document.querySelectorAll('.gallery-img');
+
+console.log(gallaryImgs);
+
+let currentlySelected = 0;
+
+prevButton.addEventListener('click', function () {
+    gallaryImgs[currentlySelected].classList.remove('active');
+    currentlySelected--; // currentlySelected = currentlySelected -1;
+    gallaryImgs[currentlySelected].classList.add('active');
+    nextButton.disabled = false;
```

Comment:
I would use smthing like

```
 element.removeAttribute("disabled");
```

above approach is explicit and self-descriptive

[READ HERE](https://stackoverflow.com/questions/7526601/setattributedisabled-false-changes-editable-attribute-to-false)